system/services/formbuilder/FormBuilderService.cfc

```diff
@@ -910,8 +910,8 @@ component {
 		}
 		for( var i=1; i <= formItems.len(); i++ ) {
 			if ( formItems[i].type.isFormField ) {
-				var columns = renderingService.getItemTypeExportColumns( formItems[i].type.id, formItems[i].configuration );
-				var export  = formItems[i].configuration.export ?: 0 ;
+				var export  = isBoolean( formItems[i].configuration.export ?: "" ) && formItems[i].configuration.export ?: "";
```

Comment:
I'm going to be really picky :) 

```
IsBoolean( somevar ?: "" ) && somevar;
// NOT:
IsBoolean( somevar ?: "" ) && somevar ?: "";
```

If `someVar ?: ""` is Boolean, then we know that it definitely exists so we don't need to the elvis operator a second time.

Reply:
Hi @DominicWatson, I removed the extra elvis operator. Thanks for the heads up.